packages/mobile/src/components/CodeInput.tsx

```diff
@@ -114,6 +119,7 @@ export default function CodeInput({
 
             {showInput ? (
               <TextInput
+                textContentType="username"
```

Comment:
oh actually let's only do this when `shortVerificationCodesEnabled` is true, right?